### PR TITLE
3212 add addition info

### PR DESCRIPTION
**Pull request by spavelo**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -191,44 +191,29 @@
...
-            Безготівкове зарахування
-        </div>
-        <div class="">Як оформлюється довіреность</div>
-        <div class="">У відділенні банку, де відкрито Ваш рахунок Ви можете оформити довіреність на право
+        <div class="addition_info">
+        <div class="title">інформація, що стане в нагоді</div>
+        <div class="subtitle">Як відбувається зарахування коштів?</div>
+        <div class="desc">
+            Готівкою через касу в день відкриття депозиту</div>
+          <div class="desc">
```
Comment (vladyslav28) [inline src/index.html]: short code should be in one line

File: src/index.html [hunk 2]
```diff
@@ -191,44 +191,29 @@
...
-                <option value=""></option>
-            </select>
-            <input type="textarea">
-            <input type="email">
-            <input type="numder">
-            <button></button>
+        <div class="subtitle">Яка здійснюється отримання процентів та повернення вкладу?</div>
+        <div class="desc">
+            Готівкою через касу </div>
+           <div class="desc">
```
Comment (vladyslav28) [inline src/index.html]: use .description

File: src/index.html [hunk 3]
```diff
@@ -191,44 +191,29 @@
...
         </div>
-        <div class="">інформація, що стане в нагоді</div>
-        <div class="">Як відбувається зарахування коштів?</div>
-        <div class="">Готівкою через касу в день відкриття депозиту
-            Безготівкове зарахування
-        </div>
-        <div class="">Як оформлюється довіреность</div>
-        <div class="">У відділенні банку, де відкрито Ваш рахунок Ви можете оформити довіреність на право
+        <div class="addition_info">
+        <div class="title">інформація, що стане в нагоді</div>
```
Comment (vladyslav28) [inline src/index.html]: Format added code before comit

Comment (spavelo) [reply src/index.html]: DONE

File: src/index.html (hunk 2 above)
Comment (spavelo) [reply src/index.html]: DONE

File: src/index.html (hunk 1 above)
Comment (spavelo) [reply src/index.html]: DONE

**Pull request merged**